technical-reference.adoc

```diff
@@ -1,512 +1,466 @@
-| Custom Headers a| Object | Custom headers to include in the request. The specified custom headers are merged with the default headers that are specified in the configuration. |  |
+| Configuration | String | `Name` of the configuration to use. | | x
+| Body a| Object | Content to use. |  `#[payload]` |
+| Config Ref a| ConfigurationProvider |  `Name` of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* <<non-repeatable-stream>> |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
+| Custom Query Parameters a| Object | Custom query parameters to include in the request. The query parameters specified here are merged with the default query parameters that are specified in the configuration. |  |
+| Custom Headers a| Object | Custom headers to include in the request. The custom headers specified here are merged with the default headers that are specified in the configuration. |  |
 | Response Timeout a| Number |  How long Mule waits for a response to complete before the request times out. |  |
 | Response Timeout Unit a| Enumeration, one of:
- 
-** NANOSECONDS
-** MICROSECONDS
-** MILLISECONDS
-** SECONDS
-** MINUTES
-** HOURS
-** DAYS |  Time unit for the *Response Timeout* field. |  |
+
+** `NANOSECONDS`
+** `MICROSECONDS`
+** `MILLISECONDS`
+** `SECONDS`
+** `MINUTES`
+** `HOURS`
+** `DAYS` |  Time unit for the *Response Timeout* field. |  |
 | Streaming Type a| Enumeration, one of:
- 
-** AUTO
-** ALWAYS
-** NEVER |  How the connector streams request content to the remote service:
- 
-* `AUTO` (default)
- 
-Connector automatically uses the best streaming strategy based on the request content.
- 
-* `ALWAYS`
- 
-Connector always streams the request content.
- 
-* `NEVER`
- 
-Connector does not stream the request content. |  |
+
+** `AUTO`
+** `ALWAYS`
+** `NEVER` |  How the connector streams request content to the remote service:
+
+* `AUTO` (default): Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`: Connector always streams the request content.
+
+* `NEVER`: Connector does not stream the request content. |  |
 | Target Variable a| String |  Name of the variable that stores the operation's output. |  |
 | Target Value a| String |  Expression that evaluates the operation's output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
-| Reconnection Strategy a| * <<Reconnect>>
-* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  |
 |===
- 
+
 === Output
- 
+
 [%autowidth.spread]
 |===
-|Type |Any
-| Attributes Type a| <<HttpResponseAttributes>>
+|Type | Object
```

Comment:
@rui-lomba, why is this empty?

Reply:
I think we are just supposed to say what is the Output Type.
Followed examples from other connectors -> https://docs.mulesoft.com/salesforce-connector/10.13/salesforce-connector-reference#output-42